### ISSUE TITLE
Accessibility for Form Results

### ISSUE THREAD

**Issue by r-sauers** [r-sauers/contact-congress-plugin#58]
I was doing some digging and found that form results should have the [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live) attribute so that screen readers will announce the form results.

None of the form results have this implemented.

**Comment by r-sauers** [comment]
Putting this as low-priority for the sole reason that the organization I'm building this for currently has no people who need screen readers.

**Comment by r-sauers** [comment]
This may be more appropriate: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-errormessage

**Comment by r-sauers** [comment]
And this: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-busy
